### PR TITLE
bump pg major

### PR DESCRIPTION
**Pull request by talevy17**

### CODE REVIEW

File: packages/plugin-gallery/web/src/gallery-component.jsx [hunk 1]
```diff
@@ -158,17 +158,30 @@ class GalleryComponent extends PureComponent {
 
   handleFilesAdded = ({ data, error, itemIdx }, uploadBIData) => {
     const handleFileAdded = (item, error, idx) => {
+      const {
+        thumbnail: { pathname, width: thmbWidth, height: thmbHeight },
```
Comment (talevy17) [inline packages/plugin-gallery/web/src/gallery-component.jsx:162]: check if relevant (probably should delete changes)

File: packages/plugin-gallery/web/src/layout-helper.ts [hunk 2]
```diff
@@ -41,26 +42,82 @@ export const getCurrentLayout = (store: Store, t: TranslationFunction) => {
...
   return [
-    { value: 2, label: t('GalleryPlugin_Layout_Grid'), icon: LayoutGridIcon },
-    { value: 1, label: t('GalleryPlugin_Layout_Masonry'), icon: LayoutMasonryIcon },
-    { value: 0, label: t('GalleryPlugin_Layout_Collage'), icon: LayoutCollageIcon },
-    { value: 3, label: t('GalleryPlugin_Layout_Thumbnails'), icon: LayoutThumbnailsIcon },
-    { value: 9, label: t('GalleryPlugin_Layout_Slideshow'), icon: LayoutSlideshowIcon },
-    { value: 6, label: t('GalleryPlugin_Layout_Panorama'), icon: LayoutPanoramaIcon },
-    { value: 7, label: t('GalleryPlugin_Layout_Columns'), icon: LayoutColumnsIcon },
-    { value: 4, label: t('GalleryPlugin_Layout_Slides'), icon: LayoutSlidesIcon },
+    { value: GALLERY_LAYOUTS.GRID, label: t('GalleryPlugin_Layout_Grid'), icon: LayoutGridIcon },
```
Comment (talevy17) [inline packages/plugin-gallery/web/src/layout-helper.ts:45]: if label is the same in both. Can merge. icon => dropDownIcon | gridIcon

File: packages/plugin-gallery/web/lib/resize-media-url.js [hunk 3]
```diff
@@ -129,7 +129,9 @@ export const resizeMediaUrl = (
   allowWatermark = false,
   focalPoint
 ) => {
-  if (originalUrl.indexOf('base64') !== -1) {
+  originalUrl = originalUrl || item.url;
```
Comment (shaulgo) [inline packages/plugin-gallery/web/lib/resize-media-url.js:132]: Is this needed?
```suggestion
```

File: packages/plugin-gallery/web/lib/resize-media-url.js [hunk 4]
```diff
@@ -129,7 +129,9 @@ export const resizeMediaUrl = (
...
+  sharpParams = sharpParams || {};
```
Comment (shaulgo) [inline packages/plugin-gallery/web/lib/resize-media-url.js:133]: is this needed?

File: packages/plugin-gallery/web/src/toolbar/inline-buttons.tsx [hunk 5]
```diff
@@ -2,7 +2,7 @@ import { BUTTONS, PluginSettingsIcon } from 'wix-rich-content-plugin-commons';
 import { getModalStyles } from 'wix-rich-content-editor-common';
 import { Modals } from '../modals';
 import { ManageMediaIcon, UploadIcon } from '../icons';
-import { galleryLayoutsDropdown, switchLayout, getCurrentLayout } from '../layout-helper';
+import { getGalleryLayouts, switchLayout, getCurrentLayout } from '../layout-helper';
```
Comment (shaulgo) [inline packages/plugin-gallery/web/src/toolbar/inline-buttons.tsx:5]: 1. export getGalleryLayoutsDropdown/Grid

2. getGalleryLayouts with bool, only internal

**Review by shaulgo — APPROVED** [review]

**Pull request merged**